dependencies.gradle

```diff
@@ -0,0 +1,49 @@
+ext.versions = [
+
+        'minSdk'            : 21,
+        'compileSdk'        : 30,
+        'targetSdk'         : 30,
+        'sourceCompat'      : JavaVersion.VERSION_11,
+        'targetCompat'      : JavaVersion.VERSION_11,
+
+        'kotlin'           : '1.5.30',
+        'kotlinCoroutines' : '1.5.1',
+        'dagger'           : '2.38.1',
+        'timber'           : '5.0.1',
+        'work'             : '2.5.0',
+        'lifecycle'        : '2.2.0',
+        'moshi'            : '1.12.0',
+        'arch'             : '2.1.0',
+        'markwon'          : '4.6.2',
+        'arrow'            : '0.8.2',
+        'rxKotlin'         : '2.4.0',
+        'rxAndroid'        : '2.1.1',
```

Comment:
Will dependabot be able to understand and update those values?

Reply:
Yes, it should work while it is the same as setting an **ext {}**  in root _build.gradle_